README.md

```diff
@@ -379,7 +379,7 @@ String that gets passed to Google Maps [Geocoder](https://developers.google.com/
 
 #### callback

```

Comment:
@jxom I think we can still support the callback in our code (until the next major version update), but we want to encourage the users to use the Promise version, so let's remove this `callback` doc from the README. I think it would make it less confusing for the developers using this library.